themes/default/content/blog/mlops-huggingface-llm-aws-sagemaker-python/index.md

```diff
@@ -0,0 +1,131 @@
+---
+date: 2023-09-08
+title: "Deploy a Huggingface Model on AWS SageMaker with Pulumi Python IaC"
+allow_long_title: true
+meta_desc: "Guided short tutorial on starting a Pulumi Infrastructure as Code project to deploy Huggingface LLM Models on Amazon Web Services SageMaker Machine Learning Platform"
+meta_image: "meta.png"
+authors:
+    - kat-morgan
+tags:
+    - ai
+    - ml
+    - iac
+    - aws
+    - llm
+    - vllm
+    - mlops
+    - llama
+    - llama2
+    - python
+    - sagemaker
+    - huggingface
+---
+
+Welcome to another post in the Pulumi + Python MLOps #MLOpsChallenge series!
```

Comment:
I really don't like the hashtag. Its not meant to be a challenge. In the previous post I called it "AI Infrastructure as Python". We can turn that into a hashtag if we want.

Reply:
Also instead of focusing on the fact its a series, I would explain the concept or the problem we are solving. "Deploying a HF model on Sagemaker takes a bit of setup to spin up. Engineers run into these XYZ problems, here is how Pulumi solves it for you"

This way, you have a nice intro to why they should bother to try this out

Reply:
+1. Can we move away from the MLops challenge branding, which then spills into social posts and so one? The whole concept gets messy.

We are trying to accelerate engineers spinning up AI infra. It's pretty simple.

Also, i am very keen on the post explaining the concept and problem you are solving with this: It's challenging to do X, because of this and this. Here is a magical way to solve it. 
Then you show the solution and outcome

Reply:
I've taken the liberty of adding a quick draft on why this is important and why readers should care. @elistaykova & @aaronkao, please review and provide feedback.

Reply:
Can we move this type of conversation back to the campaign planning doc and associated meetings? Feedback extending beyond a single blog post content review is best facilitated on the planning end of content creation.

Reply:
proceeding with hashtag #PulumiMLOps per zoom call decision